bindgen-tests/tests/expectations/tests/macro_const.rs

```diff
@@ -5,10 +5,10 @@
     non_upper_case_globals
 )]
 
-pub const foo: &[u8; 4usize] = b"bar\0";
+pub const foo: &[u8; 4] = b"bar\0";
 pub const CHAR: u8 = 98u8;
 pub const CHARR: u8 = 0u8;
 pub const FLOAT: f64 = 5.09;
 pub const FLOAT_EXPR: f64 = 0.005;
 pub const LONG: u32 = 3;
-pub const INVALID_UTF8: [u8; 5usize] = [240u8, 40u8, 140u8, 40u8, 0u8];
+pub const INVALID_UTF8: &[u8; 5] = b"\xF0(\x8C(\0";
```

Comment:
Do you know why this value became a slice in the process and was not kept as an array?

Reply:
It's a result of simplifying the code. Why would a non-UTF-8 string be a different type than a UTF-8 string?